### PR TITLE
Updated Emergency Lockers to contain Nitrogen tanks

### PR DESCRIPTION
**Pull request by DangerRevolution**
## About the PR

Added emergency Nitrogen Tanks to emergency lockers so Slimes are able to make use of them, currently; should an area be spaced Slimes are not accounted for regarding emergency lockers and so if they do not have a nitrogen tank on them, they are doomed.

**Media**

- [X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
NitrogenTankFix

**Changelog**

:cl: Danger!Revolution!
- add: Added emergency Nitrogen Tanks
- add: Added emergency Nitrogen Tanks to Emergency Lockers

### CODE REVIEW

**Comment by Colin-Tel** [comment]
It's kind of dumb there aren't any "emergency" air/n2 tanks, because that would really come in handy here.

**Comment by DangerRevolution** [comment]
> It's kind of dumb there aren't any "emergency" air/n2 tanks, because that would really come in handy here.

I am actually working on that now; I've got a working emergency N2 tank, I'm just recolouring the emergency O2 tank sprites now :) , I'll have it done at some point this week lol

File: Resources/Prototypes/Entities/Objects/Tools/gas_tanks.yml [hunk 1]
```diff
@@ -93,6 +93,33 @@
...
+    - Belt
+    - Suit Storage
+  - type: MeleeWeapon
+    damage:
+      types:
+        Blunt: 5
+  - type: PhysicalComposition
+    materialComposition:
+      Steel: 100
+# I've tried to look into it and I can't see why setting the emergency nitrogen tank to
```
Comment (DangerRevoltion) [inline Resources/Prototypes/Entities/Objects/Tools/gas_tanks.yml:118]: still looking into this.

File: Resources/Prototypes/Entities/Objects/Tools/gas_tanks.yml [hunk 2]
```diff
@@ -93,6 +93,33 @@
...
+# type: clothing
+# type: gas tank similarly to the emergency o2 tank breaks it.
+# this is the cleanest I could do - Danger!Revolution!
+
```
Comment (DangerRevoltion) [inline Resources/Prototypes/Entities/Objects/Tools/gas_tanks.yml:122]: still looking into this.

**Review by DebugOk — CHANGES_REQUESTED** [review]
It might be better to have the oxygen and nitrogen tanks in the same or group, then tweak the probabilities a little.

As an example
Full oxygen - 0.5
Full nitrogen - 0.3
Emergency oxygen - 0.10
Emergency nitrogen - 0.10

**Comment by DangerRevolution** [comment]
Does this not cause issues by randomising the viability of emergency lockers?
My interpretation of your suggestion was to make
- emergency oxygen tank
- oxygen tank
- emergency nitrogen tank
- nitrogen tank
all mutually exclusive in the loot table?
Hence: you, as a human, could now walk up to an emergency locker and potentially find a nitrogen tank in there; creating a new problem?
or: a slime could walk up to an emergency locker and STILL not find Nitrogen despite the fix?

**Comment by DebugOk** [comment]
> Does this not cause issues by randomising the viability of emergency lockers?
> My interpretation of your suggestion was to make
> - emergency oxygen tank
> - oxygen tank
> - emergency nitrogen tank
> - nitrogen tank
> all mutually exclusive in the loot table?
> Hence: you, as a human, could now walk up to an emergency locker and potentially find a nitrogen tank in there; creating a new problem?
> or: a slime could walk up to an emergency locker and STILL not find Nitrogen despite the fix?

I can see this getting abused, but eh, should be fine as is then. It'd be less of an issue of slimes started with the proper tank

**Review by DebugOk — APPROVED** [review]

**Comment by DebugOk** [comment]
You dont need to keep merging master

**Pull request closed without merging**